src/shared/components/VideoCategoryCard/VideoCategoryCard.style.ts

```diff
@@ -0,0 +1,147 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+import { transparentize } from 'polished'
+
+import { colors, sizes, transitions } from '@/shared/theme'
+
+import { Text } from '../Text'
+
+type ColorProps = { color: string }
+type LoadingProps = { loading?: boolean }
+type VariantProps = { variantCategory?: 'default' | 'compact' }
+
+export const CoverImg = styled.div<{ bgImgUrl: string }>`
+  position: relative;
+  width: 100%;
+  height: 100%;
+  background: url(${({ bgImgUrl }) => bgImgUrl});
+  background-position: center;
+  background-size: cover;
+`
+
+export const CoverImgOverlay = styled.div`
+  transition: opacity ${transitions.timings.regular} ${transitions.easing};
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  z-index: 1;
+  background-color: ${colors.gray[800]};
+  mix-blend-mode: color;
+`
+
+export const CoverImgContainer = styled.div`
+  position: relative;
+`
+
+const hoverStyles = ({ loading, color }: LoadingProps & ColorProps) =>
+  !loading &&
+  css`
+    border-left: 0 solid ${color};
+    transform: translate(-${sizes(2)}, -${sizes(2)});
+    box-shadow: ${sizes(2)} ${sizes(2)} 0 ${color};
+  `
+
+export const Container = styled.div<ColorProps & VariantProps & LoadingProps>`
+  transition: all ${transitions.timings.regular} ${transitions.easing},
+    border ${transitions.timings.sharp} ${transitions.easing};
+  display: grid;
+  cursor: pointer;
+  border-left: 4px solid ${({ color, loading }) => (color && !loading ? color : 'transparent')};
+  background-color: ${({ loading }) => (loading ? colors.gray[900] : colors.gray[800])};
+
+  &:hover {
+    ${hoverStyles}
+
+    ${CoverImgOverlay} {
+      opacity: 0;
+    }
+  }
+`
+
+const generalStyles = ({ variantCategory }: VariantProps) =>
+  variantCategory === 'default' &&
+  css`
+    height: 228px;
+    grid-template-columns: 2fr 1fr;
+  `
+export const GeneralContainer = styled(Container)`
+  ${generalStyles}
+`
+
+export const Content = styled.div<VariantProps>`
+  display: grid;
+  grid-template-rows: auto auto 1fr;
+  padding: ${({ variantCategory }) => (variantCategory === 'default' ? sizes(8) : sizes(4))};
+`
+
+export const IconCircle = styled.div<ColorProps>`
+  display: flex;
+  margin-bottom: ${sizes(4)};
+  align-items: center;
+  justify-content: center;
+  width: ${sizes(10)};
+  height: ${sizes(10)};
+  border-radius: 100%;
+  background: ${({ color }) => transparentize(0.8, color)};
```

Comment:
Could we just apply regular color and add `opacity`? I'd like to avoid `polished` helpers where possible

Reply:
hmm no because then the child elements get transparent as well and we don't want that

Reply:
True, you would need to change the layout like the design has. It has a container that has 2 children - background and icon